api/templates.go

```diff
@@ -0,0 +1,48 @@
+package api
```

Comment:
Why this file is in API? Not doing any call to Qovery API, please move it to cmd or util folder

Reply:
True, done.